platform-hub-api/spec/controllers/me_controller_spec.rb

```diff
@@ -42,6 +42,51 @@
     end
   end
 
+  describe '#delete_identity' do
+    it_behaves_like 'authenticated' do
+      before do
+        create(:identity,
+          user: current_user,
+          external_id: '1234567',
+          external_username: test_auth_payload.preferred_username,
+          external_name: test_auth_payload.name,
+          external_email: test_auth_payload.email
+        )
+      end
+
+      it 'should delete an identity successfully if it exists for the current user' do
+        expect(Audit.count).to eq 0
+        delete :delete_identity, params: {service: 'github'}
```

Comment:
The provider `github` here just happens to match the provider of the factory object you created above; a potentially nicer way would be to do `{service: @identity.provider}` here and make sure you assign `@identity = <factory object creation>` above.